correspondence/survey2018.md

```diff
@@ -0,0 +1,46 @@
+Correspondence / Survey 2018
+============================
+
+Hey everyone, as we are coming to an end of the year, and starting to make plans for next year, we figured we would send out a quick survey to gather some information about how we did, and how we can improve.
+
+So:
+
+General
+-------
+
+* Did you attend any node school events this year?
+    * yes/no
+* Did you attend the basic workshop?
+    * yes/no
+* Did you attend the open source workshop?
+    * yes/no
+
+Locations
+----
+
+* How was Sauce Labs Venue?
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+* How was Clio as a Venue?
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+* How was Elysian Coffee Venue?
+    *  1 - 10 / NA
+    *  Additional feedback/notes
+
+*  .... maybe preference for value location? Can we do ordered list?
+    * Vancouver 1-10
+    * Burnaby 1-10
+    * Surrey 1-10
+    * Richmond 1-10
+    * Other
```

Comment:
Should we include New West and Coquitlam?

Reply:
Do we have any leads on venues there?

I know @emclaren was talking about sfu surrey at some point, and i kinda assumed richmond library might be interested after the C&C stuff.

Reply:
Ah, nope. Nothing off the top of my head.

Reply:
Btw, not a No, just was my original thinking

Reply:
Yeah, I'm good either way.